ui/admin/tests/acceptance/credential-stores-test.js

```diff
@@ -32,17 +41,198 @@ module('Acceptance | credential-stores', function (hooks) {
       type: 'project',
       scope: { id: instances.scopes.org.id, type: 'org' },
     });
+    instances.credentialStore = this.server.create('credential-store', {
+      scope: instances.scopes.project,
+    });
     // Generate route URLs for resources
     urls.globalScope = `/scopes/global/scopes`;
     urls.orgScope = `/scopes/${instances.scopes.org.id}/scopes`;
     urls.projectScope = `/scopes/${instances.scopes.project.id}`;
     urls.credentialStores = `${urls.projectScope}/credential-stores`;
+    urls.credentialStore = `${urls.credentialStores}/${instances.credentialStore.id}`;
+    urls.unknownCredentialStore = `${urls.credentialStores}/foo`;
+    urls.newCredentialStore = `${urls.credentialStores}/new`;
+    // Generate resource counter
+    getCredentialStoresCount = () => {
+      return this.server.schema.credentialStores.all().models.length;
+    };
+    authenticateSession({});
   });
 
   test('visiting /credential-stores', async function (assert) {
-    assert.expect(1);
+    assert.expect(2);
     await visit(urls.credentialStores);
     await a11yAudit();
     assert.equal(currentURL(), urls.credentialStores);
+    await visit(urls.credentialStore);
+    await a11yAudit();
+    assert.equal(currentURL(), urls.credentialStore);
+  });
+
+  // This test is returning an issue regarding not finding resource.
+  // It is a false positive.
+  test.skip('visiting an unknown credential store display 404 message', async function (assert) {
```

Comment:
@randallmorey I am skipping this test because I do believe there is an issue causing a false positive. Host catalog has the same issue. Perhaps we should re-think this test?

Reply:
Let me jump in and take a look!  Thanks for catching this.